akka-http2-support/src/main/scala/akka/http/impl/engine/http2/AlpnSwitch.scala

```diff
@@ -0,0 +1,121 @@
+package akka.http.impl.engine.http2
```

Comment:
configure copyright header in your IDE please 😉 
Or... we add the license header plugin?